libraries/AP_Common/tests/test_bitmask.cpp

```diff
@@ -75,5 +88,3 @@ TEST(Bitmask, Assignment)
 }
 
 AP_GTEST_MAIN()
-
-int hal = 0; // bizarrely, this fixes an undefined-symbol error but doesn't raise a type exception.  Yay.
```

Comment:
I'd prefer to keep this in and NOT include the HAL header / pull in the symbol

Reply:
Doesn't compile without the hal ;-P